### PR TITLE
Added Numerical Python to NumPy def and typos corrected

### PR DESCRIPTION
**Pull request by anwarMZ**
A few typos and grammatical gaps corrected for clarity, Writing Numerical Python can help a learner to have context why its called NumPy and that its major work is with numerical manipulation

Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact Kate Hertweck (k8hertweck@gmail.com).  

---

### CODE REVIEW

File: _episodes/01-numpy.md [hunk 1]
```diff
@@ -791,7 +791,7 @@ tells Python how big to make this space. Each subplot is placed into the figure
 its `add_subplot` [method]({{ page.root }}/reference/#method). The `add_subplot` method takes 3
 parameters. The first denotes how many total rows of subplots there are, the second parameter
 refers to the total number of subplot columns, and the final parameter denotes which subplot
-your variable is referencing (left-to-right, top-to-bottom). Each subplot is stored in a
+your variable is referencing to (left-to-right, top-to-bottom). Each subplot is stored in a
```
Comment (maxim-belkin) [inline _episodes/01-numpy.md]: please undo this change.

File: _episodes/01-numpy.md [hunk 2]
```diff
@@ -1000,7 +1000,7 @@ the graphs will actually be squeezed together more closely.)
 > ## Drawing Straight Lines
 >
 > In the center and right subplots above, we expect all lines to look like step functions because
-> non-integer value are not realistic for the minimum and maximum values. However, you can see
+> non-integer values are not realistic for the minimum and maximum values. However, you can see
```
Comment (maxim-belkin) [inline _episodes/01-numpy.md]: 👍

**Comment by anwarMZ** [comment]
I have edited as per asked and submitted new commit, let me know if I have done it wrong?

**Comment by maxim-belkin** [comment]
FYI: this PR was auto-closed by GitHub because the commit message contained `corrected swcarpentry#651`.

File: _episodes/01-numpy.md [hunk 3]
```diff
@@ -204,8 +204,8 @@ that can be called upon when needed.
 
 ## Loading data into Python
 In order to load our inflammation data, we need to access
-([import]({{ page.root }}/reference/#import) in Python terminology) a library called
-[NumPy](http://docs.scipy.org/doc/numpy/ "NumPy Documentation").  In general you should use this
+([import]({{ page.root }}/reference/#import) in Python terminology) a library called
```
Comment (maxim-belkin) [inline _episodes/01-numpy.md]: Please delete the space at the end of the line

File: _episodes/01-numpy.md [hunk 4]
```diff
@@ -204,8 +204,8 @@ that can be called upon when needed.
 
 ## Loading data into Python
 In order to load our inflammation data, we need to access
-([import]({{ page.root }}/reference/#import) in Python terminology) a library called
-[NumPy](http://docs.scipy.org/doc/numpy/ "NumPy Documentation").  In general you should use this
+([import]({{ page.root }}/reference/#import) in Python terminology) a library called 
+[NumPy](http://docs.scipy.org/doc/numpy/ "NumPy Documentation") which stands for Numerical Python.  In general you should use this
```
Comment (maxim-belkin) [inline _episodes/01-numpy.md]: Please wrap this line to 100-characters

**Comment by maxim-belkin** [comment]
Thanks, @anwarMZ. Closing this in favor of #652.
